Dockerfile

```diff
@@ -468,7 +470,7 @@ RUN /build/scripts/install_extensions versions > /.image_config; \
     echo "PG_AUTH_MON=\"${PG_AUTH_MON}\"" >> /.image_config; \
     echo "PGBOUNCER_EXPORTER_VERSION=\"${PGBOUNCER_EXPORTER_VERSION}\"" >> /.image_config; \
     echo "PGBACKREST_EXPORTER_VERSION=\"${PGBACKREST_EXPORTER_VERSION}\"" >> /.image_config; \
-    echo "PGAI_VERSION=\"${PGAI_VERSION}\"" >> /.image_config; \
+    if [ "${PG_MAJOR}" -gt 15 ]; then echo "PGAI_VERSION=\"${PGAI_VERSION}\"" >> /.image_config; fi; \
```

Comment:
I'd suggest not guarding this- if PGAI_VERSION isn't set, we'll get an empty version and that's good too.

Reply:
done